client/main.lua

```diff
@@ -18,6 +18,7 @@ local function takeCoral(coralIndex)
         label = Lang:t('info.collecting_coral'),
         canCancel = true,
         useWhileDead = false,
+        allowSwimming = true,
```

Comment:
At the top of server/main.lua add a dependency check to assert that the server has the minimum ox_lib version for this field https://overextended.dev/ox_lib/Modules/Version/Shared